cpp/src/Ice/UdpEndpointI.cpp

```diff
@@ -149,6 +149,21 @@ IceInternal::UdpEndpointI::datagram() const
     return true;
 }
 
+shared_ptr<EndpointI>
+IceInternal::UdpEndpointI::toPublishedEndpoint(string publishedHost) const
+{
+    return make_shared<UdpEndpointI>(
```

Comment:
Why not reuse current endpoint like in tcp, is it never possible?

Reply:
It's possible, but requires more code/complexity, and there is no benefit since we don't compute these published endpoints often.